.golangci.yml

```diff
@@ -38,17 +38,6 @@ run:
 issues:
   exclude-use-default: false
   exclude-rules:
-    - path: '_test.go'
-      linters:
-        - errcheck                      # Errors may be ignored in tests.
```

Comment:
In order to simplify the reviewing process, it makes sense to split this large PR into multiple smaller PR that can be individually reviewed and merged. For example, a linter could be enabled at a time in a PR along with the changes needed for enabling that specific linter.
That is opposed to "including" all linters at once, and all the fixes for all the linters in a single larger PR.

It may still be possible to enable multiple linters at once if they do not require any code changes.

Reply:
@julio-lopez ok sure